packages/sdk-core/tasks/build-types.sh

```diff
@@ -15,5 +15,11 @@ fi
 # copy the typechain files over from ethereum-contracts
 cp -r ../ethereum-contracts/typechain-types ./src/typechain-types
 
+# Remove the Address export from typechain-types to avoid conflict with mappedSubgraphTypes
+# OpenZeppelin v5 added a custom error to Address library, giving it a non-empty ABI
+# This causes typechain to generate types for it, but it's not needed for the SDK
+sed -i '/export type { Address } from ".\/@openzeppelin\/contracts\/utils\/Address";/d' ./src/typechain-types/index.ts
+sed -i '/export { Address__factory } from ".\/factories\/@openzeppelin\/contracts\/utils\/Address__factory";/d' ./src/typechain-types/index.ts
+
```

Comment:
This became an issue with OZ5 because the Address library now contains a custom error, thus the ABI is nomore empty, thus the hardhat typescript plugin creates a type of it, colliding with the alias to string we have defined in `packages/sdk-core/src/subgraph/mappedSubgraphTypes.ts`.